crates/biome_js_analyze/src/lint/nursery/no_secrets.rs

```diff
@@ -32,12 +46,17 @@ declare_lint_rule! {
     /// ```js
     /// const nonSecret = "hello world";
     /// ```
+    ///
+    /// ## Disclaimer
+    /// This rule is intended to catch obvious secret leaks, but for more robust detection
```

Comment:
This section repeats what other above just said. Maybe we could leave short sentence above and move exact tool recommendations and explanation here?